src/core/lib/surface/completion_queue.h

```diff
@@ -47,6 +47,16 @@ typedef struct grpc_cq_completion {
   uintptr_t next;
 } grpc_cq_completion;
 
+/// For callback CQs, the following is what is actually intended by
+/// the tag.
+namespace grpc_core {
+class CQCallbackInterface {
```

Comment:
I didn't want to call it CallbackInterface since we have internal callbacks as well, and CompletionQueueCallbackInterface is really long, so I thought CQCallbackInterface. I think it should be CQ rather than Cq since CQ is for CompletionQueue .